workflows/scrnaseq.nf

```diff
@@ -193,6 +194,9 @@ workflow SCRNASEQ {
         ch_versions = ch_versions.mix(CELLRANGER_ALIGN.out.ch_versions)
         ch_mtx_matrices = ch_mtx_matrices.mix(CELLRANGER_ALIGN.out.cellranger_out)
         ch_star_index = CELLRANGER_ALIGN.out.star_index
+        ch_multiqc_cellranger = CELLRANGER_ALIGN.out.cellranger_out.map{
+            meta, outs -> outs.findAll{ it -> it.name == "web_summary.html"}
```

Comment:
Why is this part necessary? Is this something that could be changed in the `CELLRANGER_ALIGN` module to make it play nicely with multiqc out of the box?

Reply:
It's mostly to make MultiQC more efficient, because it only gets the files it needs (the full cellranger output dir contains a lot of files)